cmd/gopay/main.go

```diff
@@ -1,7 +1,42 @@
 package main
 
-import "fmt"
+import (
+	"log"
+	"net/http"
+
+	"github.com/gopay/internal"
+)
+
+func initDB() {
+	// temp fake db for accounts
+	internal.Accounts["0001"] = &internal.Account{
+		AccountId: "0001",
+		Name:      "Shankar",
+		LastName:  "Nakai",
+	}
+
+	internal.Accounts["0002"] = &internal.Account{
+		AccountId: "0002",
+		Name:      "Jessica",
+		LastName:  "Lourenco",
+	}
+
+	internal.Accounts["0003"] = &internal.Account{
+		AccountId: "0003",
+		Name:      "Caio",
+		LastName:  "Henrique",
+	}
+
+	internal.Accounts["0004"] = &internal.Account{
+		AccountId: "0004",
+		Name:      "Karina",
+		LastName:  "Domingues",
+	}
+}
 
 func main() {
-	fmt.Println("Hello... is there anybody in there?")
```

Comment:
For this type of print information it would be better to use https://github.com/rs/zerolog
It is out of scope for this PR IMO, but I would update the code base with the zerolog, this library is very used in production level project, and is very popular in the Golang community

Reply:
okk 
but why not fmt?

Reply:
actually... I removed this line
weird you saw this